### PR TITLE
slight overhaul of the shared errors section

### PR DESCRIPTION
**Pull request by daschl**
ping @ingenthr

### CODE REVIEW

File: content/sdk/shared/couchbase-errors.dita [hunk 1]
```diff
@@ -34,24 +34,30 @@
...
+           <p>If the maximum content size is larger than 20MB the server responds with an error noting that it can't store it since it is
+             too big. This error is not transient and must be raised up the stack since it likely indicates an application error that
+             creates too large document contents.</p>
         </section>
         <section id="devguide-transient-errors">
             <title>Transient and resource errors</title>
             <p>These errors may be received because of resource starvation.</p>
             <p><b>Temporary failure</b></p>
-            <p>This item is received during mutations when the cluster node has run out of memory.
+            <p>This error is received when the server is temporarily inhibiting an error that doesn't allow it to respond
```
Comment (simonbasle) [inline content/sdk/shared/couchbase-errors.dita]: "inhibiting": did you mean "exhibiting"?

File: content/sdk/shared/couchbase-errors.dita [hunk 2]
```diff
@@ -34,24 +34,30 @@
...
+              successfully. For example during mutations when the cluster node has run out of memory or is currently warming up.
                 Its disk and replication queues are full and must wait until items in those queues
                 are stored and replicated before it can begin receiving new operations.</p>
             <p>While this condition is rare, it may happen under massively concurrent writes from
                 clients and a limited memory allocation on the server.</p>
             <p>The short-term corrective action for this error is to throttle and slow down the
-                application, giving Couchbase some time for pending operations to complete before
+                application, giving Couchbase some time for pending operations to complete (or to complete warmup) before
```
Comment (simonbasle) [inline content/sdk/shared/couchbase-errors.dita]: would rewrite as "for pending operations (or warmup) to complete"

File: content/sdk/shared/couchbase-errors.dita [hunk 3]
```diff
@@ -6,25 +6,25 @@
...
-            <p>Data errors are errors returned by the data store because a certain data condition
+            <p>Data errors are errors returned by the server because a certain data condition
                 was not met. Data errors typically have very clear corrective paths.</p>
-            <p><b>Item not found</b></p>
-            <p>If an item is not found, then it has either not yet been created or has since been
-                deleted. It is received on retrieval (<i>get</i>) operations (get an item),
-                    <i>replace</i> operations (replace an item that already exists), and
-                    <i>remove</i> operations (delete an existing item).</p>
-            <p>If this error is received when attempting to retrieve an item, then the item should
+            <p><b>Document does not exist</b></p>
```
Comment (simonbasle) [inline content/sdk/shared/couchbase-errors.dita]: IIRC some SDKs will have exception names reflect more the "NotFound" wording than "DoesNotExist"... might want to put both in the "title"?

**Comment by ingenthr** [comment]
Rebased, reviewed, merged.

**Pull request closed without merging**